### PR TITLE
feat(pace-10): add apply cmd for data policy and don't apply upon ups…

### PR DESCRIPTION
**Pull request by trietsch**
…ert by default

### CODE REVIEW

**Review by ivan-p92 — APPROVED** [review]

File: pkg/common/constants.go [hunk 1]
```diff
@@ -38,3 +41,11 @@ const DatabaseFlagUsage = "database in the catalog"
 const SchemaFlag = "schema"
 const SchemaFlagShort = "s"
 const SchemaFlagUsage = "schema in database on catalog"
+
+const BlueprintFlag = "blueprint"
+const BlueprintFlagShort = "b"
+const BlueprintFlagUsage = "fetch a blueprint data policy from a catalog or a processing platform"
+
```
Comment (bobvandenhoogen) [inline pkg/common/constants.go:48]: Why did we put this back in? Thought we'd removed the blueprint flag in the cli altogether?

File: pkg/entity/datapolicy/datapolicy.go [hunk 2]
```diff
@@ -120,23 +144,20 @@ func readPolicy(filename string) *DataPolicy {
...
-	if len(args) != 0 {
-		return nil, cobra.ShellCompDirectiveNoFileComp
-	}
-
+func TableOrDataPolicyIdsCompletion(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	flags := cmd.Flags()
 
-	platformId, catalogId, blueprint := isBlueprint(flags)
+	blueprint := GetBoolAndErr(flags, common.BlueprintFlag)
+	log.Debugln(fmt.Sprintf("TableOrDataPolicyIdsCompletion blueprint: %v", blueprint))
```
Comment (bobvandenhoogen) [inline pkg/entity/datapolicy/datapolicy.go:151]: think this needs to be removed?

File: pkg/entity/processingplatform/processingplatform.go [hunk 3]
```diff
@@ -43,6 +41,8 @@ func IdsCompletion(_ *cobra.Command, args []string, _ string) ([]string, cobra.S
 	names := lo.Map(response.ProcessingPlatforms, func(p *DataPolicy_ProcessingPlatform, _ int) string {
 		return p.Id
 	})
+	log.Debugln(fmt.Sprintf("IdsCompletion names: %v", names))
+
```
Comment (bobvandenhoogen) [inline pkg/entity/processingplatform/processingplatform.go:45]: another log

File: pkg/common/constants.go (hunk 1 above)
Comment (trietsch) [reply pkg/common/constants.go:48]: Yes, but we forgot one thing: if you want to get a Data Policy that is actually stored in the PACE DB, you need to provide a processing platform (as it's a required field). However, the current behavior results in fetching a blueprint policy from the processing platform if specified.

File: pkg/entity/datapolicy/datapolicy.go (hunk 2 above)
Comment (trietsch) [reply pkg/entity/datapolicy/datapolicy.go:151]: Good catch. It doesn't really matter, but we had some issues with completions and therefore we added the debug statements. Will remove!.

File: pkg/entity/processingplatform/processingplatform.go (hunk 3 above)
Comment (trietsch) [reply pkg/entity/processingplatform/processingplatform.go:45]: Same, thanks.

**Pull request merged**